src/Tutor.Core/DomainModel/KnowledgeComponents/KnowledgeComponentMastery.cs

```diff
@@ -77,5 +93,12 @@ private void When(AssessmentEventAnswered @event)
 
             Mastery += kcMasteryIncrement;
         }
+
+        private void When(SoughtHelp @event)
+        {
+            /*
+             * Possibly record how many times help was sought in AeMastery?
+             */
+        }
```

Comment:
I feel like keeping a count of how many times help was requested for an AE could be useful, it seems like it would be useful for AE selection in cases where multiple AEs have the same mastery.

Reply:
It makes sense, but I would simplify the design for now. The plugin could be enhanced to show one hint at a time, but this is not something we can currently support (no time to write more detailed hints).

Reply:
I definitely agree, I will leave this method empty for now anyway (since there's really nothing that makes sense to put in it yet). The thought about possibly counting how many times the Learner asked for help was mostly just me trying to figure out if this actually fits into KcMastery and how.